### PR TITLE
override ledger config from aea-config.yaml in 'aea get-wealth'

### PR DESCRIPTION
**Pull request by marcofavorito**
## Proposed changes

override ledger config from aea-config.yaml in 'aea get-wealth'

## Fixes

n/a

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

n/a

### CODE REVIEW

File: packages/fetchai/skills/confirmation_aw1/strategy.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "developer_handle",
 ]
 DEFAULT_TOKEN_DISPENSE_AMOUNT = 100000
-DEFAULT_TOKEN_DENOMINATION = "atestfet"
+DEFAULT_TOKEN_DENOMINATION = "atestfet"  # nosec
```
Comment (marcofavorito) [inline packages/fetchai/skills/confirmation_aw1/strategy.py:37]: not related to my changes

File: packages/fetchai/skills/confirmation_aw1/skill.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ fingerprint:
   __init__.py: QmSRjWPGDeLsKZYQoBKLRG59WS7b7RYMrPNcNxDSaBFxCW
   dialogues.py: QmPo9vc8Msp8Bi5aFgg5BZas1tZqn7K1FgeiwMu1Sqai2Z
   handlers.py: QmaR9AgwankLMjsBGmtUkHNKymujkfoG7TkcDG45v9ut12
-  strategy.py: Qmc1UGa8abWhkmVFmJr9rBmWDD3D82eMiWiAoMsmHhUcYb
+  strategy.py: QmWNmGNj68mXRUMDsbRXBJqwtE1cNA4MErF23XbXrXYNN6
```
Comment (marcofavorito) [inline packages/fetchai/skills/confirmation_aw1/skill.yaml:13]: not related to my changes

File: packages/hashes.csv [hunk 3]
```diff
@@ -54,7 +54,7 @@ fetchai/skills/aries_alice,QmZkAfGzoXi2HQiASrESEbbpN99qAXsw3sWUyXYiEHCh1S
 fetchai/skills/aries_faber,QmVZwHEepJWA6oHwxVywVK1PQ5hXMdrYqEtf8xJ84dSCqw
 fetchai/skills/carpark_client,QmSBJG8orVcuHw3UsaCKPpXyqNfPaHtkB8gKc5AZ35U4VA
 fetchai/skills/carpark_detection,QmVPAz6qNU1c5dn2YVUBhz5RFcx6rbCECkjGjZE1bsYKvi
-fetchai/skills/confirmation_aw1,Qmbdg2w99CDreoAVRFTdDuxfuQ3u4t7uYS3gFKyWk6Mguq
+fetchai/skills/confirmation_aw1,QmNzjSE6hhnW7utYL64hjJG4JdT634ZkjT6WZcoep9nKay
```
Comment (marcofavorito) [inline packages/hashes.csv:57]: not related to my changes

File: aea/helpers/base.py [hunk 4]
```diff
@@ -336,12 +346,17 @@ def recursive_update(to_update: Dict, new_values: Dict) -> None:
         value_to_update = to_update[key]
         value_type = type(value)
         value_to_update_type = type(value_to_update)
-        if value_type != value_to_update_type and value is not None:
+        both_are_dict = _is_dict_like(value) and _is_dict_like(value_to_update)
+        if (
+            not both_are_dict
+            and value_type != value_to_update_type
+            and value is not None
+        ):
```
Comment (marcofavorito) [inline aea/helpers/base.py:354]: this needed to handle the case when we have `dict` and `OrderedDict`

File: aea/helpers/base.py [hunk 5]
```diff
@@ -336,12 +346,17 @@ def recursive_update(to_update: Dict, new_values: Dict) -> None:
...
             raise ValueError(
                 f"Trying to replace value '{value_to_update}' with value '{value}' which is of different type."
             )
 
-        if value_type == value_to_update_type == dict:
+        if both_are_dict:
```
Comment (marcofavorito) [inline aea/helpers/base.py:359]: see above

**Review by DavidMinarsch — COMMENTED** [review]
LGTM

**Pull request merged**